packages/safe-tools-client/app/components/block-explorer-button/index.gts

```diff
@@ -38,9 +41,12 @@ export default class BlockExplorerButton extends Component<Signature> {
       @as="anchor"
       @size="extra-small"
       @kind={{@kind}}
-      href={{this.blockExplorerUrl}} 
+      @href={{this.blockExplorerUrl}}
+      @disabled={{not @transactionHash}}
       target="_blank"
       rel="noopener"
+      class={{cn block-explorer-button-disabled=(not @transactionHash)}}
```

Comment:
The class on this button when disabled should be:

`block-explorer-button block-explorer-button--disabled`

In BEM, `block-explorer-button` is a "block" and we want to modify it (make a new variant) for some special case. That's when the "modifier" class is added (`block-explorer-button--disabled`).